saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md

```diff
@@ -21,9 +21,9 @@
     - Provides a non-AWS user with temporary AWS access by linking users Active Directory credentials
     - Uses SAML (Security Assertion markup language)
     - Allows Single Sign On (SSO) which enables users to log in to AWS console without assigning IAM credentials
-  - ***Federation with third party providers / Cognito***
-    - Used mainly in web and mobile applications
-    - Makes use of Facebook/Google/amazon etc. to federate them
+  - ***Federation with third party providers (Cognito & Web Identity Federation)***
+    - Cognito: Used mainly in web and mobile applications
+    - Web Identity Federation: Makes use of Facebook/Google/Amazon etc. to federate them
```

Comment:
Great attention to detail and pretty good to mention Web Identity Federation here. Awesome!

Reply:
👍